iphone/Classes/TiUITabGroup.m

```diff
@@ -96,9 +96,21 @@ - (void)handleDidShowTab:(TiUITabProxy *)newFocus
 	NSMutableDictionary * event = [NSMutableDictionary dictionaryWithCapacity:4];
 
 	NSArray * tabArray = [controller viewControllers];
-
-	NSInteger previousIndex = -1;
-	NSInteger index = -1;
+   
+    NSInteger previousIndex = 0;
```

Comment:
check indentations. If this part of code is using tab, please use tab instead of 4 spaces. you can change this in the Xcode preferences.